README.md

```diff
@@ -960,8 +960,10 @@ ConnectOptions connectOptions = new ConnectOptions()
                                         .setSocketTimeout(50)
                                         .setConnectionTimeout(50)
                                         .setMaxConnections(100)
-                                        .setProxyHost("http://localhost")
+                                        .setProxyHost("localhost")
                                         .setProxyPort(8080)
+                                        .setProxyUser("exampleProxyUser")
+                                        .setProxyPassword("examplePassword")
```

Comment:
Out of pedantry, i guess that should be `"exampleProxyPassword`" to be consistent